.gitlab-ci.yml

```diff
@@ -0,0 +1,3 @@
+include:
```

Comment:
Please remove this file from the PR.

Reply:
My bad - the corresponding commit wasn't supposed to land in github. Removed.